emsdk.py

```diff
@@ -1736,8 +1736,9 @@ def is_active(self):
         debug_print(str(self) + ' is not active, because key="' + key + '" does not exist in .emscripten')
         return False
 
-      # If running in embedded mode, all paths are stored dynamically relative to the emsdk root, so normalize those first.
-      dot_emscripten_key = dot_emscripten[key].replace("' + emsdk_path + '", emsdk_path())
+      # If running in embedded mode, all paths are stored dynamically relative
+      # to the emsdk root, so normalize those first.
+      dot_emscripten_key = dot_emscripten[key].replace("emsdk_path + '", "'" + emsdk_path())
```

Comment:
this still looks weird to me even with the explanation from before... it's shifting a `'` from the right to the left. That seems really surprising.

Can't this just replace `' emsdk_path '` (note the space before and after) with the path?

Reply:
No, we also want to replace the ` +=` part.

Old:

```
FOO = emsdk_path + "/some/path"
```

New:

```
FOO = "/the/actual/path/some/path"
```


I can probably come up with a cleaner way to do this overall, but for now this fixes the immediate breakage and I don't think its any nicer before of after this change, right?

Reply:
I see better now, thanks!